gdal/doc/source/api/gdalogrin_python.rst

```diff
@@ -0,0 +1,229 @@
+.. _gdalogrin_python:
+
+================================================================================

```

Comment:
You would strongly benefit from resynchronizing the outdated content of this page with swig/python/README.rst which was updated recently

Reply:
Fix with gdal\swig\python\README.rst